pennylane/ops/op_math/controlled_class.py

```diff
@@ -99,6 +112,20 @@ class Controlled(SymbolicOp):
 
     """
 
+    # pylint: disable=no-self-argument
+    @operation.classproperty
+    def __signature__(cls):  # pragma: no cover
+        # this method is defined so inspect.signature returns __init__ signature
```

Comment:
Why was this required?

Reply:
`inspect.signature` first looks at `__signature__`, then it looks at the call signature for `__new__`, then it looks at the call signature for `__init__`.

Since we are using `__new__` to add in `Operation` functionality, it has an altered call signature.  We don't want to use that in the documentation. We want to use `__init__`'s call signature instead.